package.json

```diff
@@ -7,6 +7,8 @@
     "postinstall": "touch node_modules/.bin/.keep"
   },
   "dependencies": {
+    "@auth0/auth0-spa-js": "^1.6.5",
+    "@babel/runtime": "^7.9.2",
```

Comment:
I needed babel/runtime (prod and dev) and babel/polyfill to support the Auth0 SDK.